dist/stylesheets/reset.css

```diff
@@ -1,36 +1,45 @@
 html {
-  box-sizing: border-box; }
```

Comment:
The files in the dist folder should not be edited directly.
Adding additional lines to auto-compiled files also adds to their size so please revert this.

Reply:
My reset.css file has that row, so not sure why it shows up here. Pls check e8f95252e6126079cd7e122d5778a952bcbf9177.